scripts/sample_maps.py

```diff
@@ -42,27 +43,31 @@
 flags.DEFINE_string("gaussian_path", "data/massivenu/mnu0.0_Maps10_PS_theory.npy", "Path to Massive Nu power spectrum.")
 flags.DEFINE_boolean("gaussian_only", False, "Only use Gaussian score if yes.")
 flags.DEFINE_boolean("reduced_shear", False, "Apply reduced shear correction if yes.")
+flags.DEFINE_integer("map_size", 360, "Map size")
```

Comment:
Do we need a map_size parameter? We can deduct it from the shear map or the mask file I guess

Reply:
Yes it can be deduced from the shear map or mask shape... I'll edit the file

Reply:
done :)